modules/sno-adding-worker-nodes-to-sno-clusters.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: PROCEDURE
+[id="sno-adding-worker-nodes-to-sno-clusters_{context}"]
+= Adding worker nodes using the Assisted Installer on console.redhat.com
+
+You can add worker nodes to {sno} clusters that were created on link:https://console.redhat.com[console.redhat.com] using the Assisted Installer.
```

Comment:
Suggesting to add the link to Assisted Installer: link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[Assisted Installer]